extensions/README.md

```diff
@@ -33,15 +35,38 @@ can be aware of it.
-   For example, the `eo:bands` attribute may be used in Item Properties to describe the aggregation of all bands available in
-   the Item Asset objects contained in the Item, but may also be used in an individual Item Asset to describe only the bands available in that asset.
-3. Additional attributes relating to a [Catalog](../catalog-spec/catalog-spec.md) or
-   [Collection](../collection-spec/collection-spec.md) should be added to the root of the object.
+## Using Extensions
+
+When deciding how to model data in STAC it is highly recommended to first look at the [list of 
+extensions](https://stac-extensions.github.io/) and re-use fields there instead of creating your own version. This
+increases interoperability, as users know that the meaning of your fields is the same as in other STAC 
+implementations. Many clients will also understand more mature extensions for better display and querying. 
+
+To incorporate an extension in STAC the 'extension ID' of the extension must be added to the `stac_extensions`
+array of the STAC [Catalog](../catalog-spec/catalog-spec.md#stac_extensions), 
+[Collection](../collection-spec/collection-spec.md#stac_extensions) or [Item](../item-spec/item-spec.md#stac_extensions)
+object. This identifier is a link to the JSON Schema URL that validates the fields in the extension, so STAC validators
+can fetch the Schema to validate that the STAC object properly follows the extension. These JSON Schema URLs also act as 
+identifiers for specific version of the extension that the STAC object implements. The extension ID can be
+found listed as the 'identifier' in the second line of the README of any extension made with the [extension 
+template](https://github.com/stac-extensions/template), and new ones get published automatically with any release made 
+with the template.
+
+### Extension IDs in `stac_extensions`
+
+The logic for when an object should list an extension ID in its `stac_extension` array is as follows:
+
+- If the object directly implements the extension (by including at least one of the fields of the extension, plus any 
+  additional specified requirements), the  `stac_extensions` of that object should contain the extension ID.
+- If an Asset implements fields of the extension, then `stac_extensions` of the Item or Collection which holds that
+  Asset should contain the extension ID.
+- If a Collection [summary](../collection-spec/collection-spec.md#summaries) contains Item fields that implement an extension, then
+  the `stac_extensions` array of that Collection should list the extension ID. For example, if a Collection `summaries` field
+  contains a summary of `eo:bands`, then that Collection should have the EO extension JSON Schema URL in the `stac_extensions` array.
+- If an object implements an extension that results in fields from a separate extension to be referenced, then the latter extension
```

Comment:
Maybe add that this is usually the case, but extensions may specify otherwise (i.e. they say explicitly that stac_extensions should not be populated?)